### ISSUE TITLE
Remember last pasted clip highlighted

### ISSUE THREAD

**Issue by wisemike2** [sabrogden/Ditto#431]
There used to be an option to remember the last pasted clip highlighted (so one can just press "Enter" directly when opening the last used clip), but seems it is missing in newer version, or I removed it somehow..

Now when I click on a clip to paste, when I open Ditto menu again, the first sticky clip is the one is highlighted, not the last pasted, which I need.

When using Ctrl+C, it remembers the selected clip highlighted, how can I keep the same after pasting?

Any help much appreciated, thanks.

**Comment by ctsmith2** [comment]
This would also help with accidentally deleting the top sticky clip when
individually selecting clips to delete.

**Comment by Chealer** [comment]
@ctsmith2: Ticket #508 discusses preventing deletion of sticky clips.

@wisemike2: Please indicate at least 1 affected version. As far as I can see, 03.25.113 always remembers its selection, except when pasting a single clip, without using _Special Paste_ → _Paste, Don't Change Clip Order_.

Does this persist if you set advanced option _Update Clip Order On Paste_ to FALSE (in General)?

I suspect that both this and issue #304 are the result of the changes to `ProcessPaste.cpp` and/or `QPasteWnd.cpp` in commit e5c5aee7aaee9555b61d6d4693b7cfb4e8133771. I suspect the effect with sticky clips is accidental.

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>

**Comment by ctsmith2** [comment]
No longer having problem. Please close this ticket.

**Comment by wisemike2** [comment]
@Chealer 
You are right about this issue not present when: setting advanced option Update Clip Order On Paste to FALSE.  But is there a way to have both? so it updates the order, and still highlight the last used clip?

**Comment by Chealer** [comment]
Thank you @wisemike2 
As this is an ITS, each ticket should emphasize the problem, in particular in the title. Please indicate at least 1 affected version and retitle with a short description of the issue. I suggest:
> Quick Paste: Preselected item is not the latest pasted (when there are sticky clips and clip order is updated)
